peripherals/sdio/sdio_f446.yaml

```diff
@@ -0,0 +1,258 @@
+_modify:
+  SDMMC:   # renaming block to be RM0390-compliant
+    name: SDIO
+    description: 'Secure digital input/output SD/SDIO MMC card host interface'
+    groupName: SDIO
+    addressBlock:
+      size: "0x400"     
+
+SDIO:
+  _strip: SDMMC_
+  _delete:
+    - '*ACKTIMER'
+    - '*IDMA*'
+    - '*VER'
+    - '*ID'
+  _modify:
+    SDMMC_POWER:
+      description: 'SDIO power control register'
+    SDMMC_CLKCR:
+      description: 'CLKCR register controls the SDIO_CK output clock.'
+    SDMMC_CMDR:
+      name: "SDMMC_CMD"  # NOTE: something weird happens on leaving just 'CMD' -- 'MD' -- ?
+    SDMMC_RESP4R:
+      addressOffset: "0x20"
+      name: RESP4
+    SDMMC_RESP3R:
+      addressOffset: "0x1C"
+      name: RESP3
+    SDMMC_RESP2R:
+      addressOffset: "0x18"
+      name: RESP2
+    SDMMC_RESP1R:
+      addressOffset: "0x14"
+      name: RESP1
+    SDMMC_RESPCMD:
+      addressOffset: "0x10"
+    SDMMC_ARGR:
+      name: ARG
+    SDMMC_RESPCMDR:
+      name: RESPCMD
+    SDMMC_DLENR:
+      name: "SDMMC_DLEN"
+    SDMMC_DCNTR:
+      name: 'SDMMC_DCOUNT'
+    SDMMC_STAR:
+      name: SDMMC_STA
+    SDMMC_MASKR:
+      name: SDMMC_MASK
+    SDMMC_FIFOR:
+      addressOffset: "0x80" # acc. to CMSIS headers (??)
+      name: SDMMC_FIFO
+  # What is the exact logic of [svdpatch.py]? Why is stripping AFTER delete & modify?
+  _add:
+    _interrupts:
+      SDIO:
+        description: SDIO global interrupt
+        value: 49
+    FIFOCNT:
+      description: 'The SDIO_FIFOCNT register contains the remaining number of words to be written to or read from the FIFO. The FIFO counter loads the value from the data length register (see SDIO_DLEN) when the data transfer enable bit, DTEN, is set in the data control register (SDIO_DCTRL register) and the DPSM is at the Idle state. If the data length is not word-aligned (multiple of 4), the remaining 1 to 3 bytes are regarded as a word.'
+      addressOffset: "0x48"
+      size: "0x20"
+      resetValue: "0x00000000"
+      access: read-only
+      fields:
+        FIFOCOUNT:
+          description: 'Remaining number of words to be written to or read from the FIFO'
+          bitOffset: 0
+          bitWidth: 24
+          access: read-only
+#    FIFOARR01R:
+#      description: 'fifo arr'
+#      addressOffset: "0x84", ..., "0x88"
+#      size: "0x20"
+#      resetValue: "0x00000000"
+#      access: read-write
+#      fields:
+#        FIFODATA:
+#          bitOffset: 0
+#          bitWidth: 32
+#          access: read-write
+#  _array:
+#    FIFOARR*: {}
+#    "RESP?R":
+#      name: "RESP%s"
+#      _modify:
+#        CARDSTATUS1:
+#          name: CARDSTATUS
```

Comment:
How come these are all commented out?

Reply:
The CARDSTATUS bits were already implemented in sdio_f4.yaml; and I'm not sure how to implement FIFO array and whether it's even needed, since most of C code I've seen sticked with just the first 32-bit word (at 0x80).